From 6eadea9387b171202c453b21726308904a38a823 Mon Sep 17 00:00:00 2001
From: Bin Yang <yangbin@rock-chips.com>
Date: Fri, 26 Oct 2018 20:44:47 +0800
Subject: [PATCH] arm64: dts: rockchip: update usb configs for
 rk3399pro-evb-v10

1. disabled bq25700 node
2. add type-c1/host0/host1 support

Change-Id: If245dd187640674c407cc8ed5e0de3b8c19d93e5
Signed-off-by: Bin Yang <yangbin@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399pro-evb-v10.dts   | 36 +++++++++++++++++--
 1 file changed, 33 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
index df3c5bef8ca7..7ec5505e3e9b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-evb-v10.dts
@@ -607,8 +607,6 @@
 			};
 
 			vcc5v0_usb: SWITCH_REG1 {
-				regulator-always-on;
-				regulator-boot-on;
 				regulator-min-microvolt = <5000000>;
 				regulator-max-microvolt = <5000000>;
 
@@ -707,7 +705,7 @@
 		ti,otg-current = <500000>;
 		ti,input-current = <500000>;
 		pd-charge-only = <0>;
-		status = "okay";
+		status = "disabled";
 	};
 };
 
@@ -869,6 +867,10 @@
 	status = "okay";
 };
 
+&tcphy1 {
+	status = "okay";
+};
+
 &tsadc {
 	rockchip,hw-tshut-mode = <1>; /* tshut mode 0:CRU 1:GPIO */
 	rockchip,hw-tshut-polarity = <1>; /* tshut polarity 0:LOW 1:HIGH */
@@ -889,6 +891,19 @@
 	};
 };
 
+&u2phy1 {
+	status = "okay";
+
+	u2phy1_otg: otg-port {
+		status = "okay";
+	};
+
+	u2phy1_host: host-port {
+		phy-supply = <&vcc5v0_usb>;
+		status = "okay";
+	};
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_xfer &uart0_cts>;
@@ -899,19 +914,34 @@
 	status = "okay";
 };
 
+&usb_host1_ehci {
+	status = "okay";
+};
 &usb_host0_ohci {
 	status = "okay";
 };
 
+&usb_host1_ohci {
+	status = "okay";
+};
+
 &usbdrd3_0 {
 	extcon = <&fusb0>;
 	status = "okay";
 };
 
+&usbdrd3_1 {
+	status = "okay";
+};
+
 &usbdrd_dwc3_0 {
 	status = "okay";
 };
 
+&usbdrd_dwc3_1 {
+	status = "okay";
+};
+
 &vopb {
 	assigned-clocks = <&cru DCLK_VOP0_DIV>;
 	assigned-clock-parents = <&cru PLL_CPLL>;
-- 
2.35.3

